posts/formalizing-isocrystals-in-lean.md

```diff
@@ -0,0 +1,211 @@
+---
+author: 'Robert Y. Lewis, Heather Macbeth'
+category: 'New in mathlib'
+date: 2022-07-19 19:30:08 UTC+02:00
+description: ''
+has_math: true
+link: ''
+slug: classification-of-one-dimensional-isocrystals
+tags: ''
+title: Classification of one-dimensional isocrystals
+type: text
+---
+
+Last year, there was a [big mathlib refactor](https://leanprover-community.github.io/blog/posts/semilinear-maps/) to replace linear maps throughout the library with  *semilinear maps*,
```

Comment:
```suggestion
Last year, there was a [big mathlib refactor](link://filename/semilinear-maps.md) to replace linear maps throughout the library with  *semilinear maps*,
```
which ensures that the links don't break if we rename the md (which I assume would give a build error) or move the blog root url.

Reply:
Local testing implies it needs the full path:
```suggestion
Last year, there was a [big mathlib refactor](link://filename/posts/semilinear-maps.md) to replace linear maps throughout the library with  *semilinear maps*,
```